Index: SConstruct
===================================================================
--- SConstruct	(révision 2615)
+++ SConstruct	(copie de travail)
@@ -627,6 +627,12 @@
     # does not remove 'lib' and 'include'
     prefixes.insert(0,os.path.dirname(os.path.normpath(env['BOOST_INCLUDES'])))
     prefixes.insert(0,os.path.dirname(os.path.normpath(env['BOOST_LIBS'])))
+    # try to find in custom cflags/ldflags also
+    for prefix in env['CUSTOM_CXXFLAGS'].split()+env['CUSTOM_LDFLAGS'].split():
+        prefix = prefix.replace('-I', '').replace('-L', '')
+        fp = os.path.abspath(os.path.dirname(os.path.normpath(prefix)))
+        if not fp in prefixes:
+            prefixes.insert(0,fp) 
     for searchDir in prefixes:
         libItems = glob(os.path.join(searchDir, LIBDIR_SCHEMA, '%s*.*' % search_lib))
         if not libItems:
@@ -876,14 +882,18 @@
     env['LIBPATH'] = ['#src']
     
     # set any custom cxxflags to come first
-    env.Append(CXXFLAGS = env['CUSTOM_CXXFLAGS'])
+    if 'CUSTOM_CXXFLAGS' in env:
+        env.Append(CFLAGS = env['CUSTOM_CXXFLAGS'])
+        env.Append(CXXFLAGS = env['CUSTOM_CXXFLAGS'])
+    # at build time, compilation directories take precedence over user custom cflags/ldflags
+    if 'CUSTOM_LDFLAGS' in env:
+        env.Append(LINKFLAGS = env['CUSTOM_LDFLAGS'])
     
     # Solaris & Sun Studio settings (the `SUNCC` flag will only be
     # set if the `CXX` option begins with `CC`)
     SOLARIS = env['PLATFORM'] == 'SunOS'
     env['SUNCC'] = SOLARIS and env['CXX'].startswith('CC')
     
-    
     # If the Sun Studio C++ compiler (`CC`) is used instead of GCC.
     if env['SUNCC']:
         env['CC'] = 'cc'
